### PR TITLE
Add version information to ReadMe

### PR DESCRIPTION
**Pull request by GabyUnalaq**
There should be a section in the ReadMe that specifies what version should you use with Ankaios.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,14 +6,14 @@
 
 # Ankaios Python SDK for Eclipse Ankaios
```
Comment (windsource) [inline README.md:7]:
```suggestion
# Python SDK for Eclipse Ankaios
```

Having Ankaios two times does not sound that good.

File: README.md [hunk 2]
```diff
@@ -6,14 +6,14 @@
...
-Eclipse Ankaios provides workload and container orchestration for automotive
+[Eclipse Ankaios](https://github.com/eclipse-ankaios/ankaios) provides workload and container orchestration for automotive
 High Performance Computers (HPCs). While it can be used for various fields of
 applications, it is developed from scratch for automotive use cases and provides
 a slim yet powerful solution to manage containerized applications.
 
 The Python SDK provides easy access from the container (workload) point-of-view
 to manage the Ankaios system. A workload can use the Python SDK to run other workloads
-and get the state of the Ankaios system. 
+and get the state of the Ankaios system.
```
Comment (windsource) [inline README.md:16]: What about: "A workload can use the Python SDK to start, stop and update other workloads and configs and get the state of the Ankaios system."

File: README.md [hunk 3]
```diff
@@ -40,6 +40,15 @@ pip install -e ".[dev]"
 > **Note:**  
 > Depending on your Linux distribution, it could be that you need to create and activate a [virtual environment](https://docs.python.org/3/library/venv.html) to run the pip commands.
 
+## Versions
+
+Please make sure the Python SDK is compatible with the version of Ankaios you are using. For information regarding versioning, please refer to this table:
+
+| Ankaios    | Python SDK |
+| -------- | ------- |
+| 0.4.x and bellow | no sdk available |
```
Comment (windsource) [inline README.md:49]:
```suggestion
| 0.4.x and below | No Python SDK available. Please update Ankaios. |
```

**Review by windsource — CHANGES_REQUESTED** [review]
See comments for review findings.

File: README.md [hunk 4]
```diff
@@ -40,6 +40,15 @@ pip install -e ".[dev]"
 > **Note:**  
 > Depending on your Linux distribution, it could be that you need to create and activate a [virtual environment](https://docs.python.org/3/library/venv.html) to run the pip commands.
 
+## Versions
```
Comment (windsource) [inline README.md:43]:
```suggestion
## Compatibility
```

File: README.md (hunk 1 above)
Comment (GabyUnalaq) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (GabyUnalaq) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (GabyUnalaq) [reply README.md]: Done.

File: README.md (hunk 4 above)
Comment (GabyUnalaq) [reply README.md]: Done.

**Review by krucod3 — APPROVED** [review]
LGTM 👍

**Pull request merged**